### PR TITLE
[IMP] Inventory: Update dropshipping doc

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR replaces the version 16 screenshots in the dropshipping doc with screenshots from version 17. In addition, some of the language has been updated to match what appears in the new screenshots.

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @ksc-odoo,

This PR is ready for the first round of review.

Thanks,
John

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst [hunk 1]
```diff
@@ -59,16 +59,16 @@ a sales order to view the associated :abbr:`RFQ (Request for Quotation)`.
 
 Once the :abbr:`RFQ (Request for Quotation)` is confirmed, it becomes a purchase order and a
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst:60]:
```suggestion
Once the :abbr:`RFQ (Request for Quotation)` is confirmed, it becomes a purchase order, and a
```
OPTIONAL added comma for enhanced sentence clarity

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst [hunk 2]
```diff
@@ -59,16 +59,16 @@ a sales order to view the associated :abbr:`RFQ (Request for Quotation)`.
...
 dropship receipt is created and linked to it. The receipt can be viewed by clicking the
-:guilabel:`Receipt` smart button in the top right corner of the purchase order form.
+:guilabel:`Dropship` smart button in the top right corner of the purchase order form.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst:62]:
```suggestion
:guilabel:`Dropship` smart button in the top-right corner of the purchase order form.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst [hunk 3]
```diff
@@ -59,16 +59,16 @@ a sales order to view the associated :abbr:`RFQ (Request for Quotation)`.
...
    :align: center
    :alt: A dropship purchase order with the Receipt smart button in the top right corner.
 
-The dropship receipt will list the supplier in the :guilabel:`Receive From` section and the customer
-in the :guilabel:`Destination Location` section. Upon delivery of the product to the customer, click
-on the :guilabel:`Validate` button at the top left of the dropship receipt to confirm the delivered
-quantity.
+The dropship receipt displays :guilabel:`Partners/Vendors` in the :guilabel:`Source Location` field,
+and :guilabel:`Partners/Customers` in the :guilabel:`Destination Location` field. Upon delivery of
+the product to the customer, click on the :guilabel:`Validate` button at the top left of the
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst:70]:
```suggestion
the product to the customer, click on the :guilabel:`Validate` button at the top-left of the
```

**Review by ksc-odoo — APPROVED** [review]
Hey @hojo-odoo -- just finished my Peer Review of this PR. Looks good. Just have a few minor suggestions that require your attention. Once you implement the necessary changes, feel free to tag this for Final Review. Thanks! 👍

**Comment by hojo-odoo** [comment]
Hey @StraubCreative,

This PR is ready for final review.

Thanks,
John

**Review by StraubCreative — APPROVED** [review]
Hi @hojo-odoo 
Changes look good.

A few suggestions, while we're here on this doc:
- Consider renaming the doc to something that more fits h1 guidelines, like **Dropshipping** (current title is long + redundant)
- Under the `enable-dropship-route.png` image it might be worth adding an admonition block explaining what happens when the **Buy** route is also selected alongside **Dropship** (if anything at all 😉 ). A lot of content out there (from us, partners, etc.) shows both **Buy** and **Dropship** routes selected but doesn't actually explain it, so we have an opportunity to address it clearly here in the docs.
- Line 45: the intro sentence is worded strangely (doesn't have to be online purchase for example)
- The **tip** on line 51 becomes redundant by rewording line 45.

Free to move to Tech Review when you're ready, thanks!

**Comment by hojo-odoo** [comment]
Hey @samueljlieber,

This PR is ready for tech review.

Thanks,
John

**Review by samueljlieber — APPROVED** [review]
Hi @hojo-odoo! Thanks for your updates to this Dropshipping doc, your changes look good to me. Thank you for your work! Approving 👍
..
@robodoo r+

**Pull request closed without merging**